cpp-client-reference/test/test_cxx11.cpp

```diff
@@ -0,0 +1,3 @@
+#include "volition.h"
+
+auto main() -> int {}
```

Comment:
i mean these files dont seem to useful.

Reply:
For now, their existence ensures the C header contains code that is also legal C++, as this source file is compiled as C++11. I plan to add some example code (should be mostly the same as the C version).